app/config-default.js

```diff
@@ -60,7 +60,10 @@ module.exports = {
     env: {},
 
     // set to false for no bell
-    bell: 'SOUND'
+    bell: 'SOUND',
+
+    // set to false for no copy-on-select
```

Comment:
the comment should be more descriptive? Like "If `true`, selected text will automatically be copied to the clipboard.